packages/integrations/useFocusTrap/index.ts

```diff
@@ -0,0 +1,126 @@
+import { MaybeElementRef, unrefElement, tryOnUnmounted, Fn } from '@vueuse/core'
+import { watch, ref, Ref } from 'vue-demi'
+import { ActivateOptions, createFocusTrap, DeactivateOptions, FocusTrap, Options } from 'focus-trap'
+
+export interface UseFocusTrapOptions extends Options {
+  /**
+   * Immediately activate the trap
+   */
+  immediate?: boolean
+}
+
+export interface UseFocusTrapReturn {
+  /**
+   * Indicates if the focus trap is currently active
+   */
+  hasFocus: Ref<boolean>
+
+  /**
+   * Indicates if the focus trap is currently paused
+   */
+  isPaused: Ref<boolean>
+
+  /**
+   * Activate the focus trap
+   *
+   * @link https://github.com/focus-trap/focus-trap#trapactivateactivateoptions
+   * @param opts Activate focus trap options
+   */
+  activate: (opts?: ActivateOptions) => void
+
+  /**
+   * Deactivate the focus trap
+   *
+   * @link https://github.com/focus-trap/focus-trap#trapdeactivatedeactivateoptions
+   * @param opts Deactivate focus trap options
+   */
+  deactivate: (opts?: DeactivateOptions) => void
+
+  /**
+   * Pause the focus trap
+   *
+   * @link https://github.com/focus-trap/focus-trap#trappause
+   */
+  pause: Fn
+
+  /**
+   * Unpauses the focus trap
+   *
+   * @link https://github.com/focus-trap/focus-trap#trapunpause
+   */
+  unpause: Fn
+}
+
+/**
+ * Reactive focus-trap
+ *
+ * @link https://vueuse.org/integrations/useFocusTrap/
```

Comment:
```suggestion
 * @link https://vueuse.org/useFocusTrap
```

We have HTTP alias set in Netlify